tools/SchemaManager.Core/SqlSchemaManager.cs

```diff
@@ -273,14 +266,14 @@ private async Task ValidateVersionCompatibility(int maxAvailableVersion, Cancell
         }
     }
 
-    private async Task<string> GetScriptAsync(int version, string scriptUri, CancellationToken cancellationToken, string diffUri = null)
+    private async Task<string> GetScriptAsync(bool applyFullVersion, int version, CancellationToken cancellationToken)
```

Comment:
Is this method necessary? Could we not simply call the `_schemaClient` methods directly?

Reply:
Good catch! Definitely not necessary. Fixed in latest commit.